daemon/pool.go

```diff
@@ -32,6 +36,50 @@ var (
 	errDriverStopped = errors.NewKind("driver stopped")
 )
 
+var (
+	policyDefaultWindow    = mustEnvDur("BBLFSHD_POLICY_WINDOW", time.Second*3)
+	policyDefaultTick      = mustEnvDur("BBLFSHD_POLICY_TICK", time.Millisecond*100)
+	policyDefaultMin       = mustEnvInt("BBLFSHD_POLICY_MIN", 1)
+	policyDefaultScale     = mustEnvInt("BBLFSHD_POLICY_SCALE_INC", 1)
+	policyDefaultDownscale = mustEnvFloat("BBLFSHD_POLICY_DOWNSCALE_MULT", 0.25)
```

Comment:
Although all these are all private, but as soon as they are exposed though evn vars - I'm sure the next maintainer, as well as any person doing Ops, will be very grateful for any documented tips on what each of those are for.